src/harness/unittests/typingsInstaller.ts

```diff
@@ -519,32 +521,32 @@ namespace ts.projectSystem {
             const commander = {
                 path: "/a/data/node_modules/@types/commander/index.d.ts",
                 content: "declare const commander: { x: number }",
-                typings: "@types/commander"
+                typings: typingsName("commander")

```

Comment:
I edited this on linux, so it's surprising to see CRLF being added. Especially only on the lines that I happened to change. (If I accidentally changed the file's line endings, I would expect to see the whole thing change.)